sdk/eventhub/event-hubs/src/eventHubReceiver.ts

```diff
@@ -751,25 +754,25 @@ export class EventHubReceiver extends LinkEntity {
           options = this._createReceiverOptions(receiverOptions);
         }
 
-        log.error(
+        log.info(
           "[%s] Trying to create receiver '%s' with options %O",
           this._context.connectionId,
           this.name,
           options
         );
         this._receiver = await this._context.connection.createReceiver(options);
         this.isConnecting = false;
-        log.error(
+        log.info(
           "[%s] Receiver '%s' with address '%s' has established itself.",
           this._context.connectionId,
           this.name,
           this.address
         );
-        log.receiver(
+        log.info(
```

Comment:
The above 2 log statements should be combined and the one that appears right after (talking about options) should be verbose